EIPS/eip-6596.md

```diff
@@ -0,0 +1,717 @@
+        },
+        "copyrightRightDocumentURI": {
+          "type": "string",
+          "description": "Link to legal document outlining the rights of the token owner. Specific dimensions include the right to display a work via digital and physical mediums, present the work publicly, create or sell copies of the work, and create or sell derivations from the underlying asset.",
+          "format": "uri"
+        },
+        "isOwnerAssetCopyrightRightHolder": {
+          "type": "boolean",
+          "description": "Flags whether or not the HAT token holder is the copyright owner of the underlying asset. Does not include the copyright to the underlying asset"
+        },
+        "assetCopyrightRightDocumentURI": {
+          "type": "string",
+          "description": "Link to legal document outlining the rights of the token owner. Specific dimensions include the right to display a work via digital and physical mediums, present the work publicly, create or sell copies of the work, and create or sell derivations from the HAT.",
+          "format": "uri"
+        },
+        "isOwnerTokenReprintRightHolder": {
+          "type": "boolean",
+          "description": "Flags whether or not the HAT token owner has non-exclusive reprint/second serial rights to the HAT. Does not include the rights to the underlying asset"
+        },
+        "tokenReprintRightDocumentURI": {
+          "type": "string",
+          "description": "Link to legal document outlining the reprint/second serial rights of the token owner"
+        },
+        "isOwnerAssetReprintRightHolder": {
+          "type": "boolean",
+          "description": "Flags whether or not the HAT token owner has non-exclusive reprint/second serial rights to the underlying asset. Does not include the rights to the underlying asset"
+        },
+        "assetReprintRightDocumentURI": {
+          "type": "string",
+          "description": "Link to legal document outlining the reprint/second serial rights of the token owner"
+        },
+        "isEligibleForRelatedTokenAirdrops": {
+          "type": "boolean",
+          "description": "Flags whether or not the HAT token holder is eligible for related HAT airdrops"
+        },
+        "isEligibleForMetaverseAccess": {
+          "type": "boolean",
+          "description": "Flags whether or not the HAT token holder can gain access to the HAT metaverse"
+        }
+      },
+      "required": [
+        "assetType",
+        "id",
+        "issuers",
+        "summary",
+        "issueDate",
+        "editionNumber",
+        "editionCount",
+        "fileURI",
+        "fileSize",
+        "fileFormat",
+        "latestPossibleCreationDate"
+      ]
+    }
+  },
+  "required": [
+    "name",
+    "description",
+    "image",
+    "properties"
+  ]
+}
+```
+
+For [ERC-1155](./eip-1155.md) smart contracts, the full Metadata JSON Schema is extended to be:
```

Comment:
Same here; You shouldn't provide the full metadata when combined, only the extension

Reply:
This has been addressed in commit  e8c48f5.